### ISSUE TITLE
NotBeApproximately

### ISSUE THREAD

**Issue by jnyrup** [fluentassertions/fluentassertions#596]
There does not seem to be support for the negation of `BeApproximately(expectedValue, precision)`.

I propose addition of `NotBeApproximately(unexpectedValue, precision)`.

If the proposal is welcomed, I'll be happy to submit a PR once 5.0 is merged into master (unless somebody beats me to it).

**Comment by dennisdoomen** [comment]
See #594
